File: Java Script - Introduction to Programming/4.2. Complex Conditions - Exam Problems/03. Operations.js
function operations([x, y, operator]) {
  const sum = eval(x + operator + y)
  if (sum === Infinity || sum !== sum) return `Cannot divide ${x} by zero`
  else if (operator == "/") return `${x} ${operator} ${y} = ${sum.toFixed(2)}`
  else if (operator == "%") return `${x} ${operator} ${y} = ${sum}`
  else return `${x} ${operator} ${y} = ${sum} - ${(sum & 1) == 0 ? 'even' : 'odd'}`
}

//console.log(operations([10, 12, '+']))

File: Pyrhon - Introduction to Programming/4.2. Complex Conditions - Exam Problems/03. Operations.py
n_one, n_two, operator = int(input()), int(input()), input()
try:
    result = eval(f'n_one  {operator}  n_two')
    if operator == '+' or operator == '-' or operator == '*':
        is_even  = 'even' if result % 2 == 0 else 'odd'
        print(f'{n_one} {operator} {n_two} = {result} - {is_even}')
    elif operator == '/':
        print(f'{n_one} {operator} {n_two} = {result:.2f}')
    else:
        print(f'{n_one} {operator} {n_two} = {result}')
except:
    print(f'Cannot divide {n_one} by zero')



'''
7
3
*
#7 * 3 = 21 - odd
'''
